translator/tocwconfig/totomlconfig/tomlConfigTemplate/tomlConfig.go

```diff
@@ -241,6 +241,7 @@ type (
 	// Output plugins
 
 	cloudWatchOutputConfig struct {
+		PersistentQueue     bool   `toml:"persistentQueue"`
```

Comment:
I thought this change was for logs. Why are we including this in the CloudWatch metrics config too?

Reply:
My bad, panda, I will change it right now.